### DISCUSSION TITLE
`write_html()` list item bullets rendered in red

### DISCUSSION THREAD

**Discussion by ckamps** [py-pdf/fpdf2#1056] [General]
Perhaps similar to this issue, #880, I'm finding that bullets associated list items are rendered in red.

Given this example:

```python
from fpdf import FPDF

pdf = FPDF()
pdf.add_page()
pdf.write_html("""An unordered list:<ul>
  <li>apple</li>
  <li>banana</li>
  <li>orange</li>
</ul>""")
pdf.output("write_html_ul_example.pdf")
```
The bullets are rendered in red:

<img width="360" alt="image" src="https://github.com/py-pdf/fpdf2/assets/1332810/f341514f-d695-459d-b874-1fdb2aee9ad8">

Is the following code responsible for this setting? If so, can this code be modified to honor `pdf.set_text_color()` as was apparently done in support of #880 ?

https://github.com/py-pdf/fpdf2/blob/2.7.5/fpdf/html.py#L412C37-L412C47

**Comment by Lucas-C** [comment]
Hi @ckamps, and thank you for reporting this

Sadly, as you noticed, this red color is hacdoded in `fpdf/html.py`: https://github.com/py-pdf/fpdf2/blob/master/fpdf/html.py#L512
Fun fact, it is like that since the initial version of `fpdf2`, forked from PyPDF: https://github.com/reingart/pyfpdf/blob/master/fpdf/html.py#L207

We should really make this configurable.
Would you be interested in contributing a PR?

> can this code be modified to honor `pdf.set_text_color()` as was apparently done in support of https://github.com/py-pdf/fpdf2/discussions/880 ?

Yes, text color should be honored by default.
As we already support a `ul_bullet_char`, we could also support an optional `ul_bullet_color` setting.

By the way, we also have a hardcoded color set for `<blockquote>` elements 😞 : https://github.com/py-pdf/fpdf2/blob/master/fpdf/html.py#L498

**Comment by Lucas-C** [comment]
I opened PR https://github.com/py-pdf/fpdf2/pull/1126 to implement this
